File: 628. Maximum Product of Three Numbers/solution.java
class Solution {
  public int maximumProduct(int[] nums) {
    Arrays.sort(nums);
    int firstResult = nums[0] * nums[1] * nums[nums.length - 1];
    int secondResult = nums[nums.length - 1] * nums[nums.length - 2] * nums[nums.length - 3];
    if (firstResult > secondResult){
      return firstResult;
    }
    return secondResult;
  }
}

File: 628. Maximum Product of Three Numbers/solution.js
/**
 * @param {number[]} nums
 * @return {number}
 */
 var maximumProduct = function(nums) {
  nums.sort(function(a,b){return a - b});
  let firstResult = nums[0] * nums[1] * nums[nums.length - 1];
  let secondResult = nums[nums.length - 1] * nums[nums.length - 2] * nums[nums.length - 3];
  if (firstResult > secondResult){
    return firstResult;
  }
  return secondResult;
};